scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,58 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi /var/lib/tftpboot
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi /var/lib/tftpboot /etc/tgt
+
+### Copy config/template files
+cp bmi_config.cfg.test /etc/bmi/bmiconfig.cfg
+cp ims/ipxe.temp /etc/bmi/ipxe.cfg
+cp ims/mac.temp /etc/bmi/pxelinux.cfg
+
+virtualenv .bmi_venv
+source .bmi_venv/bin/activate
+pip install python-cephlibs
```

Comment:
ok then it makes sense. ceph-common installs python-rbd and python-rados which have the cephlibs (Tested this). Since we are using virtualenv we will need it as it uses separate site-packages